discovery-provider/src/queries/query_helpers.py

```diff
@@ -1214,15 +1214,9 @@ def add_users_to_tracks(session, tracks, current_user_id=None):
 
     Returns: None
     """
-    user_ids = get_users_ids(tracks)
-    users = []
-    if tracks and len(tracks) > 0 and tracks[0].get("user"):
-        users = list(map(lambda t: t["user"][0], tracks))
-    else:
-        # This shouldn't happen - all tracks should come preloaded with their owners per the relationship
-        users = get_unpopulated_users(session, user_ids)
-        logger.warning("add_users_to_tracks() called but tracks have no users")
-    set_users_in_cache(users)
```

Comment:
not sure if this is important, but it looks like this lines functionality wasn't carried over

Reply:
this should be implicitly handled by the relationship we have between tracks and users https://github.com/AudiusProject/audius-protocol/blob/master/discovery-provider/src/models/tracks/track.py#L84

and generally should make almost all queries faster